internal/web/server.go

```diff
@@ -87,6 +97,20 @@ func (s *server) Start(ctx context.Context, support *ServerConfigSupport) error
 		return errors.New("server is already listening")
 	}
 
+	if support == nil {
+		cfg := client.ConfigFromContext(ctx)
+		isBootstrapped, err := bootstrap.IsBootstrapped(ctx, cfg)
+		if !isBootstrapped || err != nil {
+			support = &ServerConfigSupport{
+				BootstrapMissing:    !isBootstrapped,
+				BootstrapCheckError: err,
+			}
+		}
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "\nFailed to check whether Glasskube is bootstrapped: %v\n\n", err)
+		}
+	}
+
 	s.support = support
 	s.ctx = ctx
```

Comment:
Ok interesting, I haven't really thought about the contexts yet. Maybe I'll change it right in this PR.

Reply:
Now I see what you mean, I would repair/refactor the context storing in #140 because there we will need to reload the config and reinitialize some things, I think that could be a good fit.